core/templates/dev/head/pages/exploration-editor-page/services/exploration-states.service.spec.ts

```diff
@@ -144,4 +148,24 @@ describe('ExplorationStatesService', function() {
       });
     });
   });
+
+  describe('.getSolicitAnswerDetailsMemento', function() {
```

Comment:
Ah, let's not create separate `describe` block, instead create new `it` block and the message in `it` block should start with `should ...` (`should set the solicit answer details value correctly`.

In general, the message in `describe` block + message in `it` block should create a complete sentence. 

e.g: `ExplorationStatesService should set the solicit answer details value correctly`.

Reply:
Done